buildkite/pipelines.go

```diff
@@ -90,3 +98,73 @@ func (ps *PipelinesService) List(org string, opt *PipelineListOptions) ([]Pipeli
 
 	return *pipelines, resp, err
 }
+
+// Get a pipeline by slug for a given organization.
+//
+// buildkite API docs: https://buildkite.com/docs/api/pipelines#get-a-pipeline
+func (ps *PipelinesService) Get(org string, slug string) (*Pipeline, *Response, error) {
+	u := fmt.Sprintf("v2/organizations/%s/pipelines/%s", org, slug)
+	req, err := ps.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	pipeline := new(Pipeline)
+	resp, err := ps.client.Do(req, pipeline)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return pipeline, resp, err
+}
+
+// Create a pipeline for a given organization.
+//
+// buildkite API docs: https://buildkite.com/docs/api/pipelines#create-a-pipeline
+func (ps *PipelinesService) Create(org string, p *Pipeline) (*Pipeline, *Response, error) {
+	u := fmt.Sprintf("v2/organizations/%s/pipelines", org)
+	req, err := ps.client.NewRequest("POST", u, p)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	pipeline := new(Pipeline)
+	resp, err := ps.client.Do(req, pipeline)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return pipeline, resp, err
+}
+
+// Edit a pipeline by slug for a given organization.
+//
+// buildkite API docs: https://buildkite.com/docs/rest-api/pipelines#update-a-pipeline
+func (ps *PipelinesService) Edit(org string, slug string, p *Pipeline) (*Pipeline, *Response, error) {
```

Comment:
I think this should be `Update`, unless there is prior art for `Edit` in the other endpoints.

Reply:
I don't think there were any `Update` or `Edit` endpoints anywhere in this client yet, and personally I prefer `Update` too, but I was going off what `go-github` does 🤷‍♂️